lib/task/import/validate/CsvEventValuesValidator.class.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+/*
+ * This file is part of the Access to Memory (AtoM) software.
+ *
+ * Access to Memory (AtoM) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Access to Memory (AtoM) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Access to Memory (AtoM).  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * CSV Multi Value test. Test if specified mulitvalue columns are all of the same length.
+ * e.g.: eventType, eventActor, etc...
+ *
+ * Issue warning if eventTypes has nonzero count of x eventTypes, and any another column
+ * has a different number of event values than eventTypes. Do not warn for empty fields.
+ *
+ * @author     Steve Breker <sbreker@artefactual.com>
+ *
+ * @internal
+ * @coversNothing
+ */
+class CsvEventValuesValidator extends CsvBaseValidator
+{
+    const TITLE = 'Event Value Count Test';
+    protected $matchList = [
```

Comment:
Cosmetic: blank line between `const` and `protected` definitions.

Reply:
Blank line added.